packages/amplify-category-api/src/graphql-transformer/transformer-options-v2.ts

```diff
@@ -284,6 +284,7 @@ const generateTransformParameters = (
     ),
     sandboxModeEnabled,
     enableTransformerCfnOutputs: true,
+    allowDestructiveGraphQLSchemaUpdates: false,
```

Comment:
Since managed table is used only for CDK construct, the API category will always have false value for the attribute